packages/docusaurus/src/webpack/plugins/CleanWebpackPlugin.ts

```diff
@@ -168,7 +168,7 @@ export default class CleanWebpackPlugin {
 
     if (this.checkIfFileNameIsDuplicate()) {
       throw new Error(
-        `Output directory ${this.outputPath} already exists. Docusaurus needs this directory to save the build output. Either remove the directory or choose a different build directory via '--out-dir'.`,
+        `Output file ${this.outputPath} already exists. Docusaurus needs this directory to save the build output. Either remove/change the file or choose a different build directory via '--out-dir'.`,
```

Comment:
"Output file build already exists" is not a good error message IMHO

There is no concept of "output file" in Docusaurus.


My suggestion:

`A file '${this.outputPath}' already exists. Docusaurus needs this directory to save the build output. Either remove/change the file or choose a different build directory via '--out-dir'.`,